File: android/src/main/java/com/reactnativehypertracksdk/common/GeotagData.kt
package com.reactnativehypertracksdk.common

import android.location.Location

/**
 * The data that represents geotag to create
 *
 * @param data Geotag payload
 */
internal data class GeotagData(
    val data: Map<String, Any?>,
    val expectedLocation: Location?
)

File: ios/common/GeotagData.swift
import HyperTrack

struct GeotagData {
    let data: [String: Any]
    let expectedLocation: HyperTrack.Location?

    init(data: [String: Any], expectedLocation: HyperTrack.Location?) {
        self.data = data
        self.expectedLocation = expectedLocation
    }
}
